tools/ampir/ampir.xml

```diff
@@ -0,0 +1,85 @@
+<tool name="ampir" id="ampir" version="0.01">
```

Comment:
Done.

Reply:
in the wrong place :( Fixed after a reminder. Turns out to require galaxyxml PR to allow profile parameter to the `<command> `so it's a hack until I get the energy for that plus the test. Can fix the replacement for stdio at the same time so that will be good.